runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -2562,6 +2564,111 @@ J9::Z::PrivateLinkage::buildVirtualDispatch(TR::Node * callNode, TR::RegisterDep
+      // Shape of the ccData
+      // snippetOrCompiledMethod -> to be updated in binary encoding while emitting snippet body
+      // ramMethod -> Filled Here
+      TR::MethodSymbol *methodSymbol = callSymRef->getSymbol()->castToMethodSymbol();
+      intptr_t ramMethod;
+#if defined(J9VM_OPT_JITSERVER)
+      ramMethod = comp()->isOutOfProcessCompilation() && methodSymbol->isInterpreted() ?
+                     static_cast<intptr_t>(callSymRef->getSymbol()->castToResolvedMethodSymbol()->getResolvedMethod()->getPersistentIdentifier()) :
+                     static_cast<intptr_t>(methodSymbol->getMethodAddress());
+#else
+      ramMethod = reinterpret_cast<intptr_t>(methodSymbol->getMethodAddress());
+#endif
+      ccStaticSpecialDataAddress->ramMethod = ramMethod;
+      callSnippetCCDataAddress = reinterpret_cast<intptr_t>(ccStaticSpecialDataAddress);
+      offsetOfSnippetOrCompiledMethod = static_cast<int32_t>(offsetof(ccUnresolvedStaticSpecialData, snippetOrCompiledMethod));
+      }
+
+   TR::LabelSymbol *snippetLabel = generateLabelSymbol(cg());
+   TR::LabelSymbol *doneLabel = generateLabelSymbol(cg());
+   TR::Snippet *callSnippet = new (trHeapMemory()) TR::S390J9CallSnippetRX(cg(),
+                                                                           callNode,
+                                                                           snippetLabel,
+                                                                           doneLabel,
+                                                                           callSymRef,
+                                                                           argSize,
+                                                                           callSnippetCCDataAddress);
+   void *targetAddress = reinterpret_cast<void *>(callSnippetCCDataAddress + static_cast<intptr_t>(offsetOfSnippetOrCompiledMethod));
+   TR::StaticSymbol *snippetOrCompiledSlotSymbol = TR::StaticSymbol::createWithAddress(comp()->trHeapMemory(), TR::Address, targetAddress);
+   TR::SymbolReference *snippetOrCompiledSlotSymRef = new (comp()->trHeapMemory()) TR::SymbolReference(comp()->getSymRefTab(), snippetOrCompiledSlotSymbol, 0);
+   snippetOrCompiledSlotSymbol->setNotDataAddress();
+
+   TR::RegisterDependencyConditions *preDeps = new (trHeapMemory()) TR::RegisterDependencyConditions(dependencies->getPreConditions(), NULL,
+                                                                                                      dependencies->getAddCursorForPre(), 0, cg());
+
+   int32_t numOfAdditionalPostDeps = 0;
+   TR::Register *regRA = dependencies->searchPostConditionRegister(getReturnAddressRegister());
+   if (regRA == NULL)
+      {
+      regRA = cg()->allocateRegister();
+      numOfAdditionalPostDeps+=1;
+      }
+
+   TR::Register *regEP = dependencies->searchPostConditionRegister(getEntryPointRegister());
+   if (regEP == NULL)
+      {
+      regEP = cg()->allocateRegister();
+      numOfAdditionalPostDeps+=1;
```

Comment:
Spaces around operators.

Reply:
Changed in https://github.com/eclipse/openj9/compare/9a9d9b37daeb983c8d4e295fcc785afeecde5d78..de725983cd06ff96468a76b30321d3f8bb48caea